addons/GME/Scripts/Game/GME/Editor/Modules/GME_Modules_Base.c

```diff
@@ -22,15 +31,32 @@ class GME_Modules_Base : GenericEntity
 	{
 		super.EOnInit(owner);
 		
-		if (!GetGame().InPlayMode() && !Replication.IsServer())
+		if (!GetGame().InPlayMode() || !Replication.IsServer())
```

Comment:
So we return if game is not in playmode, or we are not the server? 
But before both had to be false. Is this to ensure it isn't run on workbench before clicking play?

Reply:
`EOnInit` runs in the workbench as well as in play mode (see https://community.bistudio.com/wiki/Arma_Reforger:Entity_Lifecycle). So because I made a mistake here, it was still run in the workbench as well as on clients...

Reply:
ah. Good catch!